Containerfile.cert-manager.acmesolver

```diff
@@ -1,4 +1,4 @@
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.22 AS builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.23 AS builder
```

Comment:
Please update the go version to `1.23` in the go.mod too.

Reply:
Done.